src/micromamba/main.cpp

```diff
@@ -599,6 +602,18 @@ install_specs(const std::vector<std::string>& specs, bool create_env = false, in
     MSolver solver(pool, { { SOLVER_FLAG_ALLOW_DOWNGRADE, 1 } });
     solver.add_jobs(create_options.specs, SOLVER_INSTALL);
 
+    if (!create_options.no_pin)
+    {
+        solver.add_pins(file_pins(prefix_data.path() / "conda-meta/pinned"));
```

Comment:
this is not cross-platform. 

Use `...path() / "conda-meta" / "pinned"`

Reply:
nice catch, thanks!